policyengine_us/parameters/gov/usda/snap/income/deductions/utility/limited/allowance.yaml

```diff
@@ -1,184 +1,235 @@
-description: A household is eligible for the Limited Utility Allowance if it does not have separate heating and cooling costs, but does have at least two other utilities. Also known as the 'basic' utility allowance.
+description: A household is eligible for the Limited Utility Allowance if it does
+  not have separate heating and cooling costs, but does have at least two other utilities.
+  Also known as the 'basic' utility allowance.
 AK:
   2010-01-01: 0
   2021-10-01: 0
+  2023-10-01: 0.0
```

Comment:
I dont think we need to values if they do not change as they are set from that first date forward,

In this case just having 
`2010-01-01: 0 ` would be sufficient

Reply:
That approach makes sense, I have historically seem reference links used to identify the most recent publications used to determine the parameter value (e.g. in a state model where the tax rate is not adjusted). But I see the value for more complex programs such as snap

Reply:
Yeah I think it makes sense to update values when they haven't changed, if the statutory/regulatory source does so. As Pavel said, many tax parameters don't change year to year because the law hasn't changed, so we don't need to clutter updates in those cases. I'm not sure which applies in this case: Is Alaska actively deciding each year to not have a LUA, or are there laws or regs that haven't been updated? Since the source we have doesn't specify (not a primary source) I think we can repeat as Efraim has done for now.